lec04-dplyr.Rmd

```diff
@@ -1,4 +1,724 @@
 ---
-title: "Introduction to data wrangling using the dplyr R package"
+title: Data wrangling and visualization in the tidyverse
 ---
 
+
+> ### Learning Objectives
+>
+> * Understand the split-apply-combine concept for data analysis.
+> * Use `summarize`, `group_by`, and `tally` to split a data frame into groups
+> of observations, apply a summary statistics for each group, and then combine
+> the results.
+> * Produce scatter plots, line plots, and histograms using ggplot.
+> * Set universal plot settings.
+> * Understand and apply faceting in ggplot.
+> * Explore scientific questions using dplyr and ggplot.
+
+
+> ### Lesson outline
```

Comment:
I usually have an idea of approximately where to take the break but then I adjust depending on how fast the lecture progresses. I try to break around 3 pm, probably before ggplot this time, maybe after the first ggplot section. The challenges are part of their respective section rather than being explicit timings.